<launch>

    <arg name="map_name"/>
    <arg name="scene"/>
    <arg name="start_time"/>

    <node pkg="visualization_tool" type="visualization_tool.py" name="visualization_tool" output="log"/>

    <node pkg="visualization_tool" type="visualization_tool_node" name="coverage_rate" output="log">
        <param name="map_name" value="$(arg map_name)"  type="string"/>      
        <param name="scene" value="$(arg scene)"  type="string"/>   
        <param name="start_time" value="$(arg start_time)"  type="string"/>        
    </node>
</launch>